lib/jar_dependencies.rb

```diff
@@ -179,29 +178,28 @@ def local_maven_repo
                              detect_local_repository(maven_local_settings) ||
                              detect_local_repository(maven_user_settings) ||
                              detect_local_repository(maven_global_settings) ||
-                             File.join( user_home, '.m2', 'repository' )
+                             File.join(user_home, '.m2', 'repository')
     end
 
     def home
       absolute(to_prop(HOME)) || local_maven_repo
     end
 
-    def require_jars_lock!( scope = :runtime )
+    def require_jars_lock!(scope = :runtime)
       urls = jars_lock_from_class_loader
-      if urls and urls.size > 0
+      if !urls&.empty?
```

Comment:
this seems wrong ;)

Reply:
@mkristian Yeah, right? ;) But it’s [valid syntax](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/) since Ruby v2.3.0 and in JRuby since v9.1 (see "safe navigation operator" in https://github.com/jruby/jruby/issues/3479). But since I figure it will break for a Ruby language level below v2.3.0 I’ll change it back.

Reply:
sorry I did not know this syntax :) but let's stick to older syntax for older jruby